Allowable SLA fix

kollil opened the pull request:
Fixed the query to display the allowable SLA data on the protocol display page.

kollil committed:
ehrsla template created

kollil committed:
Fixed the query with correct linked schema

kollil committed:
Fixed the query with correct linked schema, take #2

labkey-keith commented:
@kollil I notice that the tests are not passing on this feature branch. Please be sure to look at these before considering merging this. Please reach out if you have questions about the cause of the error.

kollil committed:
Deleting the ehrSLA.xml file as it was created wrongly

kollil commented:
> @kollil I notice that the tests are not passing on this feature branch. Please be sure to look at these before considering merging this. Please reach out if you have questions about the cause of the error.

@keith-labkey, I have noticed the ehrsla.xml file got included into this fb, I thought this was deleted from the 22.3 final merge. I didn't dig deeper as I am busy with the other sla issues. Went ahead and deleted this file as it is irrelevant and updated the fb. Hopefully it gets deleted from the master after its deployed.
